lib/simpleanidb/models.py

```diff
@@ -16,15 +16,17 @@ def __init__(self, anidb, aid, auto_load=True, xml=None):
         self.episodes = {}
         self.picture = None
         self.rating_permanent = None
+        self.count_permanent = None
         self.rating_temporary = None
+        self.count_temporary = None
         self.rating_review = None
         self.categories = []
         self.tags = []
         self.start_date = None
         self.end_date = None
         self.description = None
 
-        if xml:
+        if len(xml):
```

Comment:
Else it will give this warning: FutureWarning: The behavior of this method will change in future versions.  Use specific 'len(elem)' or 'elem is not None' test instead.
  if xml:

Quick google recommends to use an explict len() or not None. Using len(), means the xml object has to have a minimum of one child. As the xml object should always have children, len() is also a valid check.